docs/UserGuide.md

```diff
@@ -1,134 +1,301 @@
-Format: `clear`  
-
-#### Exiting the program : `exit`
-Exits the program.<br>
-Format: `exit`  
-
-#### Saving the data 
-Address book data are saved in the hard disk automatically after any command that changes the data.<br>
-There is no need to save manually.
+###3 Find Task by Partial Keyword
+
+The find by partial keyword command enables you to search for any events, deadlines, to-dos that have been added to Simply. If the searched task is not found, an error message will be shown.
+
+> Format: **Find** &lt;keyword&gt;
+
+
+Example: 
+
+Find **siloso**
+
+Find 050316
+
+Find 2359
+
+Find #CS2103
+
+
+Only the tasks with the keyword are displayed.
+
+###4 Mark done tasks 
+
+When you are done with tasks, enter this command to mark them as done and hide them away.
+
+> Format: Done &lt;index&gt;
+
+> Format: Done &lt;index&gt;-&lt;index&gt;
+
+> Format: Done &lt;index&gt;, &lt;index&gt;
+    
+    
+Example: 
+
+Done T1
+
+Done T1-T3
+
+Done T1, T3
+
+
+>Note: if you want to select more than one task to complete, you can separated the task by a - to complete all tasks that are within the range. In addition, you also need to separate the task by a ; to individually delete them.
+
+###5 Display tasks
+
+####5.1 Display completed tasks
+
+The display command will display the completed tasks in their respective categories.
+
+> Format: Display completed
+
+####5.2 Display all tasks
+
+This is mainly used to return to the default screen showing all tasks.
+
+> Format: Display all
+
+###6 Undo the Most Recent Commands
+
+The undo command enables you to undo the most recent executed command.
+
+> Format: Undo &lt;number of operations&gt;
+
+Example: 
+
+Undo
+
+Undo 2
+
+
+>Note: The undo command can only undo a maximum of 5 executed commands
+
+>Note: If no number is entered, the default number of times the command will undo is 1.
+
+
+###7 Redo the Most Recent Commands
+
+The redo command enables you to redo the most recent undone command.
+
+> Format: Redo &lt;number of operations&gt;
+
+Example: 
+
+Redo
+
+Redo 2
+
+>Note: The Redo command can only undo a maximum of 5 undone commands
+
+>Note: If no number is entered, the default number of times the command will redo is 1.
+
+
+###8 Delete task
+
+####8.1 Deleting a task
+
+The delete command enables you to delete the tasks that you no longer need.
+
+> Format: Delete &lt;index&gt;
+
+> Format: Delete &lt;index&gt;-&lt;index&gt;
+
+> Format: Delete &lt;index&gt;, &lt;index&gt; 
+
+
+Example: 
+
+Delete T3
+
+Delete T1-T3
+
+Delete T1, T3
+
+
+>Note: The delete command is flexible and allows you to delete more than 1 task at a time. If the indexes entered are separated by a -, Simply will delete all tasks between the numbers including the numbers enter. If the indexes are separated by a , Simply will delete the tasks entered individually. 
+
+####8.2 Deleting a tag / multiple tags
+
+>Format: **Delete** &lt;index&gt; #tag1 #tag2
+
+Example: Delete E1 #win #victor #finally  
```

Comment:
How is this different from editing a tag? Looks a bit inconsistent to add this to delete.